kobo/django/upload/models.py

```diff
@@ -25,7 +26,8 @@ class FileUpload(models.Model):
     owner       = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
     name        = models.CharField(max_length=255)
     checksum    = models.CharField(max_length=255)
-    size        = models.PositiveIntegerField()
+    # models.PositiveBigIntegerField would be even better but it was introduced only in Django 3.1
+    size        = models.BigIntegerField(validators=[MinValueValidator(0)])
```

Comment:
Any idea what happens with this code if the migration hasn't been applied?  Can a model field declared as BigInteger successfully read/write on the table (assuming values are "small") if the migration is ignored?

Basically the same concern/question as https://github.com/release-engineering/kobo/pull/211#discussion_r1278645732 .

Reply:
Note, that the situation here is different than in https://github.com/release-engineering/kobo/pull/211#discussion_r1278645732.  This migration does not create a new field, it just extends the range of an exiting one.

I've tested this code without applied migrations on a local instance of OpenScanHub and it works as expected, small files still work fine and large files raise a database error:
1. Small file:
```python
$ osh/client/osh-cli mock-build --tarball-build-script=test ./small
Task info: http://0.0.0.0:8000/task/2/
Watching tasks (this may be safely interrupted)...
2 MockBuild: FREE
--> FREE: 1 [total: 1]
...
```

2. Large file:
```python
$ osh/client/osh-cli mock-build --tarball-build-script=test ./large
...
xmlrpc.client.Fault: <Fault 1: Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.DataError: integer out of range
...
```